### ISSUE TITLE
Bigger sea serpent hitbox

### ISSUE THREAD

**Issue by Spinogaming** [AlexModGuy/Ice-and-Fire-Suggestions#886]
Sea serpents should have bigger hitboxes, or have multiple hitboxes like dragons

**Comment by swordofblaze** [comment]
Agreed, its almost impossible to fight the larger sea serpents because I'm not certain where to strike.
